### PR TITLE
update spark doc to distinguish between JDBC and Native connector + c…

### PR DESCRIPTION
**Pull request by BentsiLeviav**
…ode examples + indentation

### CODE REVIEW

**Comment by BentsiLeviav** [comment]
I also add Glue documentation (that links to Spark JDBC doc)

File: docs/en/integrations/data-ingestion/apache-spark/index.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ for converting data types when reading from ClickHouse into Spark and when inser
 | `Object`                            |                      | ❌         |              |                                        |
 | `Nested`                            |                      | ❌         |              |                                        |
 
-# Spark JDBC
+# Spark JDBC
```
Comment (serprex) [inline docs/en/integrations/data-ingestion/apache-spark/index.md:427]:
```suggestion
# Spark JDBC
```

File: docs/en/integrations/index.mdx [hunk 2]
```diff
@@ -102,6 +102,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Amazon Kinesis|<KinesisSVG style={{width: '3rem', height: '3rem'}} />|Data ingestion|Integration with Amazon Kinesis.|[Documentation](/docs/en/integrations/clickpipes/kinesis/)|
 |Amazon MSK|<AmazonMskSVG style={{width: '3rem'}} />|Data ingestion|Integration with Amazon Managed Streaming for Apache Kafka (MSK).|[Documentation](/docs/en/integrations/kafka/cloud/amazon-msk/)|
 |Amazon S3|<S3SVG style={{width: '3rem', height: '3rem'}} />|Data ingestion|Import from, export to, and transform S3 data in flight with ClickHouse built-in S3 functions.|[Documentation](/docs/en/integrations/data-ingestion/s3/index.md)|
+|Amazon Glue|<img src={require('./images/logos/glue_logo.png').default} class="image" alt="Amazon Glue logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion| Query ClickHouse over JDBC |[Documentation](/docs/en/integrations/glue)|
```
Comment (serprex) [inline docs/en/integrations/index.mdx:105]:
```suggestion
|Amazon Glue|<img src={require('./images/logos/glue_logo.png').default} class="image" alt="Amazon Glue logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|Query ClickHouse over JDBC|[Documentation](/docs/en/integrations/glue)|
```

**Review by mzitnik — APPROVED** [review]

File: docs/en/integrations/index.mdx (hunk 2 above)
Comment (mzitnik) [reply docs/en/integrations/index.mdx:105]: @BentsiLeviav, just fix it and i already approved

**Pull request merged**